Rmds/r_and_rstudio.Rmd

```diff
@@ -0,0 +1,78 @@
+# R and RStudio {-}
+
+To complete the installs of R and RStudio, you must have administrator privileges on your machine. Choose the version appropriate for your operating system. You can also watch a video for a [Mac](https://youtu.be/Oy9BPxt8gQ0?t=270) or a [Windows](https://youtu.be/7f9ADV2v7CI?t=336) install (you only need to install R and RStudio, and none of the other programs mentioned in the videos). Please also note that the versions of R and RStudio in the video are older. Instead, install or update to the latest version supported by your operating system.
+
+1. R https://cran.r-project.org/
+    - version `r getRversion()` (only for macOS Yosemite or older: legacy version `r params$r_legacy_version`)
+    - Choose among "Download and Install R" section (**not** "Source Code for all Platforms").
+    
+        **Note for Mac users:** If you have an M1 (and higher) Mac, you **must** install the Intel version of R (Bioconductor packages installed below are not yet compatible with the arm64 version).
```

Comment:
Change to "M1 or later"?

Reply:
Not sure what is more correct here, I went with the language they used on the R install page.